src/ucp/core/ucp_context.c

```diff
@@ -398,7 +398,7 @@ static ucs_config_field_t ucp_config_table[] = {
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
   {"KEEPALIVE_INTERVAL", "20s",
-   "Time interval between keepalive rounds.",
+   "Time interval between keepalive rounds. Must be greater than 0.",
```

Comment:
may be "must be non-zero value"? time units may have number value, `auto` and `inf`

Reply:
fixed